src/games/strategy/engine/data/PlayerID.java

```diff
@@ -10,7 +9,7 @@
 import games.strategy.triplea.delegate.Matches;
 import games.strategy.util.CompositeMatchAnd;
 
-public class PlayerID extends NamedAttachable implements NamedUnitHolder, Serializable {
```

Comment:
Serializable is a marker interface. If this class is ever serialized, it probably should have the marker interface. Otherwise we also do not need the serial version UID